### PR TITLE
Adds new integration [mandarons/ha-bouncie]

### PR DESCRIPTION
**Pull request by mandarons**
**Link to successful HACS action:**  https://github.com/mandarons/ha-bouncie/actions/runs/3907347465
**Link to successful hassfest action (if integration):**  https://github.com/mandarons/ha-bouncie/actions/runs/3907347465

### CODE REVIEW

**Comment by ludeeus** [comment]
<https://github.com/mandarons/ha-bouncie/blob/main/.devcontainer/Dockerfile#L1>
You should move away from that ([reason](https://github.com/ludeeus/container#deprecated)).
Example for an alternative:
- [`.devcontainer.json`](https://github.com/ludeeus/integration_blueprint/blob/main/.devcontainer.json)
- [scripts dir instead of `container` CLI](https://github.com/ludeeus/integration_blueprint/tree/main/scripts)

File: integration [hunk 1]
```diff
@@ -508,6 +508,7 @@
   "mampfes/ha_freeair_connect",
   "mampfes/hacs_dwd_pollenflug",
   "mampfes/hacs_waste_collection_schedule",
+  "mandarons/ha-bouncie",
```
Comment (ludeeus) [inline integration:511]: <https://github.com/mandarons/ha-bouncie/blob/main/custom_components/bouncie/usage.py>
That seems invasive.
Please point me to the documentation for it, it was not in your README.

Comment (mandarons) [reply integration:511]: It is described in this file: https://github.com/mandarons/ha-bouncie/blob/main/USAGE.md. It is based on analytics.home-assistant.io.

Comment (ludeeus) [reply integration:511]: No one is going to stumble upon that file.
The entire logic is not needed. <https://analytics.home-assistant.io/custom_integrations.json>.
Just ask your users to enable "Usage analytics" <https://www.home-assistant.io/integrations/analytics/#usage-analytics>

Comment (mandarons) [reply integration:511]: Cool. I didn't know that HA Analytics also tracks custom integrations. I will remove the usage implementation.

**Pull request merged**